src/main/java/com/srans/nestserver/controller/VacateController.java

```diff
@@ -57,25 +56,31 @@ public Vacation saveVacationDetails(@Valid @RequestBody Vacation vacation) throw
 		return vacation;
 	}
 
-	@GetMapping("users/vacate/{id}")
-	@PreAuthorize("permitAll()")
-	public Vacation displayVacationDetails(@PathVariable(value = "id") Long tenantId) throws NSException {
-		logger.info("IN::POST::/users/vacation::displayVacationDetails::" + tenantId);
-		
-		logger.info("OUT::POST::/users/vacation::displayVacationDetails::" + tenantId);
-
-		
-		return vacateService.getVacationDetails(tenantId);
-
-	}
+	/*
+	 * @GetMapping("users/vacate/{id}")
+	 * 
+	 * @PreAuthorize("permitAll()") public Vacation
+	 * displayVacationDetails(@PathVariable(value = "id") Long tenantId) throws
+	 * NSException {
+	 * logger.info("IN::POST::/users/vacation::displayVacationDetails::" +
+	 * tenantId);
+	 * 
+	 * logger.info("OUT::POST::/users/vacation::displayVacationDetails::" +
+	 * tenantId);
+	 * 
+	 * 
+	 * return vacateService.getVacationDetails(tenantId);
+	 * 
+	 * }
+	 */
 	
-	@GetMapping("users/vacate/approved/{id}")
+	@PutMapping("users/vacate")
 	@PreAuthorize("permitAll()")
-	public String vacationApproved(@PathVariable(value="id") Long tenantId)throws NSException{
```

Comment:
Method name should be like verb, approveVaction()

Reply:
I have chenged it also..